CONTRIBUTING.md

```diff
@@ -160,19 +220,70 @@ make html -b linkcheck
 
 The finished documentation can be found in the `docs/_build/html` directory.
 
-
-## Release Checklist
-
-The following is a checklist to complete before taging a release.
-This helps us ensure everything goes smoothely before our army of robots send
-PyVista out to PyPI and Conda-Forge for deployment. Please checkout the
-latest state of the `master` branch locally and:
-
-1. Run all tests as outlined in the [Testing Section](#testing) and ensure
-all are passing.
-
-2. Ensure all builds and tests on Azure are passing for the latest commits on
-the `master` branch
+#### Creating a New Pull Request
+
+Once you have tested your branch locally, push your branch to (pyvista
+GitHub)[https://github.com/pyvista/pyvista] and create a pull request
```

Comment:
Non-members of pyvista probably can't do that. Above it's written to fork first, in which case you can push to your own repo and PR from there (CI still kicks in, at least did for me). Or if it's really wanted to have branches in the pyvista repo always, the step above to fork isn't necessary.

Reply:
Now asking that they just submit a PR to pyvista while merging to master.